src/shared/services/document-management/document-management.service.ts

```diff
@@ -29,9 +29,7 @@ export class DocumentManagementService {
       .post(url, formData, {headers, observe: 'body'})
       .pipe(delay(DocumentManagementService.RESPONSE_DELAY))
       .pipe(
-        map(response => {
-          return response;
-        })
+        map(body => body) // EUI-3495. Not sure we need this?
```

Comment:
This is suspect? Do we need a map here?

Reply:
Hence my comment question.

It's inside the `pipe()`, though, and @Daniel-Lam recently found some peculiarities around the document upload functionality so I didn't want to make a change to it at this point as it _is_ working the way it is. I simply made the code clearer for the time being by replacing `response` with `body` and removing the unnecessary code block.

For the moment, I'd prefer to leave it as it is and have another look at it at some point in the future - it _looks_ like a pointless operation, though, I agree.

Reply:
Mapping `body => body` does nothing. It's fair if there is a reason.

Reply:
Done and pushed.